### PR TITLE
Fixed Spacial Displacements

### PR DESCRIPTION
**Pull request by nesstea**
Finally...

### CODE REVIEW

**Comment by TeoTwawki** [comment]
Wrong change..Its the ID thats is wrong, not the cs events.

File: scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua [hunk 1]
```diff
@@ -1,115 +1,110 @@
...
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger
+-----------------------------------
+
+function onTrigger(player,npc)
+	local id = npc:getID();
+	local base = 16900333; -- (First Spacial Displacement in NPC_LIST)
```
Comment (starlightknight) [inline scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua]: 16900333 -> 16900334 is the only change needed here

File: scripts/zones/Riverne-Site_B01/npcs/Spatial_Displacement.lua [hunk 2]
```diff
@@ -1,128 +1,122 @@
...
-function onTrade(player,npc,trade)
-end;
-
------------------------------------
--- onTrigger
------------------------------------
-
-function onTrigger(player,npc)
-
-	local base = 16896182; -- First Spacial Displacement in NPC_LIST
```
Comment (starlightknight) [inline scripts/zones/Riverne-Site_B01/npcs/Spatial_Displacement.lua]: 16896182 -> 16896183 is the only change needed here

File: scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua (hunk 1 above)
Comment (nesstea) [inline scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua]: ok reverting and changing.

**Pull request merged**